yt_dlp/extractor/bbc.py

```diff
@@ -1255,6 +1266,38 @@ def extract_all(pattern):
                 lambda s: self._parse_json(s, playlist_id, fatal=False),
                 re.findall(pattern, webpage))))
 
+        # US accessed article with single embedded video (e.g.
+        # https://www.bbc.com/news/uk-68546268)
+        next_data = traverse_obj(self._search_nextjs_data(webpage, playlist_id), (
+            'props', 'pageProps', 'page'), get_all=False)
```

Comment:
The spec of `_search_nextjs_data()` is wrong. The `default` should be the default returned (here `{}`), not some internal default in case the sought JS is not found. Is it too late to change it?

Reply:
See #8937

Reply:
I actually committed the same problem upstream without thinking it through :-(.

Reply:
The fix for this has been committed. Merge master and use `default={}` instead